sickbeard/classes.py

```diff
@@ -212,7 +212,7 @@ def selectSeries(allSeries):
 
 
 class Proper(object):
-    def __init__(self, name, url, date, show, seeders, leechers, size, pubdate, hash, proper_tags):
+    def __init__(self, name, url, date, show, seeders, leechers, size, pubdate, torrent_hash, proper_tags):
```

Comment:
FYI "torrent_hash" is actually specifically called "info_hash" in the [Bittorrent Protocol Specification](https://wiki.theory.org/BitTorrentSpecification). Maybe we want to use that name as well, instead of a more generic and confusing "torrent_hash".